### ISSUE TITLE
FEATURE REQUEST: add WI-C600N

### ISSUE THREAD

**Issue by Nexinity7** [semvis123/Sonyfy#4]
This headset is already compatible with turning off noise cancelation and using the ambient sound control but cannot toggle active noise cancellation. Using an iPhone 6s on 14.1 btw.

**Comment by semvis123** [comment]
Is this the same issue as #1 ?
Or do you experience something else?

**Comment by Nexinity7** [comment]
sure is

**Comment by semvis123** [comment]
could you also try https://github.com/semvis123/Sonyfy/issues/1#issuecomment-835389157 
If I have that information I should be able to add the support

**Comment by Nexinity7** [comment]
absolutely, i'll get back to you later on today once i get that installed

**Comment by semvis123** [comment]
This issue will be handled in #1
